cmake/FindCython.cmake

```diff
@@ -4,7 +4,7 @@
 
 # This finds the "cython" executable in your PATH, and then in some standard
 # paths:
-FIND_FILE(CYTHON_BIN cython /usr/bin /usr/local/bin)
+FIND_PROGRAM(CYTHON_BIN cython /usr/bin /usr/local/bin)
```

Comment:
`FIND_FILE` finds a file while `FIND_PROGRAM` finds an executable. When `cython.exe` is in the path it is found by `FIND_PROGRAM`